src/main/webapp/app/pages/companyPage/CompanyPage.tsx

```diff
@@ -98,13 +105,13 @@ export default class CompanyPage extends React.Component<
   async getCompanyInfos() {
     try {
       this.getCompanyStatus = PromiseStatus.pending;
+      this.companyTableLoading = true;
+      this.userDropdownLoading = true;
       this.company = await client.getCompanyUsingGET({
         id: parseInt(this.props.match.params.id),
       });
       this.selectedLicenseStatus = this.company.licenseStatus as LicenseStatus;
-      await this.getCompanyUsers();
-      await this.getDropdownUsers();
-      await this.getAllUsersTokens();
+      this.getCompanyUsers();
```

Comment:
We should stick with `remoteData` when you want to fetch data and uses the status.